feat(add a template for issues): provide new option for metrics (#3186)
jenkins-infra/helpdesk: README.md

- [🧩 Issue](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=triage&template=1-report-issue.yml): An issue with the Jenkins projects Infrastructure
- [🚀 GitHub permissions](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=github-permissions&template=2-github-permissions.yml): Request GitHub permissions
- [📝 Documentation](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=documentation&template=3-documentation.yml): Let us know if any documentation is missing or could be improved
- [🌐 Crowdin localization projects](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=crowdin&template=4-crowdin.yml): Request the creation of new projects on crowdin.jenkins.io (to localize a plugin) or update its members
- [💬 Community forum](https://community.jenkins.io/): Please ask and answer questions here
- [✉️ Mailing lists](https://www.jenkins.io/mailing-lists/): You can also raise a question on the infra mailing list

> - [📊 Infra Specific agent metrics](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=inframetrics&template=5-metrics.yml): Request infrastructure specific metrics on a Jenkins agent after a build

Reviewer: ```suggestion
- [📊 Infra Specific agent metrics](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=inframetrics&template=5-metrics.yml): Request infrastructure specific metrics on a Jenkins agent to monitor a build by yourself
```
(nit)

> - [📊 Infra Specific agent metrics](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=inframetrics&template=5-metrics.yml) : if you want to get information about the metrics on a specific agent after a built

Reviewer: ```suggestion
- [📊 Infra Specific agent metrics](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=inframetrics&template=5-metrics.yml): Request infrastructure specific metrics on a Jenkins agent after a build
```
What "after a build" is for there? Is it necessary?
